draft-ietf-tls-tls13.md

```diff
@@ -2957,8 +2956,17 @@ the responses).
 
 
 ### Key and IV Update {#key-update}
+     enum { update_not_requested(0), update_requested(1), (255)
```

Comment:
What is the intended behavior if an implementation sees a value other than the ones listed? (I think we should say that's fatal and we need an extension if we ever wanted to make that extensible... hopefully this doesn't happen.)

For completeness, an alternate spelling that saves a byte would be to define two messages, KeyUpdate and KeyUpdateMutual that both trigger a key update but one also implicitly requests. (I don't care between the spellings. One has this weird Boolean enum and the other makes two distinct messages have overlapping behaviors.)